sycl/source/detail/event_impl.cpp

```diff
@@ -128,6 +128,7 @@ event_impl::event_impl(RT::PiEvent Event, const context &SyclContext)
 }
 
 event_impl::event_impl(const QueueImplPtr &Queue) : MQueue{Queue} {
+  MIsProfilingEnabled = Queue->is_host() || Queue->MIsProfilingEnabled;
```

Comment:
I don't see a reason why `MIsProfilingEnabled` should ever change so you could make it `const` and make the following change:
```suggestion
event_impl::event_impl(const QueueImplPtr &Queue) : MQueue{Queue}, MIsProfilingEnabled{Queue->is_host() || Queue->MIsProfilingEnabled} {
```

Reply:
Yes, you are right, I'll change it right now.